### DISCUSSION TITLE
Request timeout/Abort signal

### DISCUSSION THREAD

**Discussion by ilyaDegtyarenko** [nuxt/nuxt#17391] [Questions]
I get an error during the request

**_error_**

`[nitro] [dev] [unhandledRejection] Error: Unknown object type "abortsignal"`

**_code example_**
```
const controller = new AbortController(),
    timerId = setTimeout(() => {
        controller.abort()
    }, 1000)

await useFetch('[url]', {
    signal: controller.signal,
    ...
})

clearTimeout(timerId)
```

What is wrong?

**Comment by Flanker72** [comment]
Seems `useFetch` have an issue passing signal.
Converting to `$fetch` solves the problem, the signal works.

**Reply by ilyaDegtyarenko** [reply]
Yes, thank you. I am currently using $fetch for this request, but wanted to use the useFetch to cache the query
